umake/tools.py

```diff
@@ -337,6 +337,15 @@ def create_launcher(desktop_filename, content):
         gsettings.set_strv("favorites", launcher_list)
 
 
+def add_exec_link(exec_path, destination_name):
```

Comment:
so, this will need small tests (I'll put the removal of links as another utility in tools.py as well), and add of course some inline documentation